app/src/features/surveys/view/survey-animals/SurveyAnimalsTableActions.tsx

```diff
@@ -59,22 +62,26 @@ const SurveyAnimalsTableActions = (props: ITableActionsMenuProps) => {
           <ListItemIcon>
             <Icon path={mdiPlus} size={1} />
           </ListItemIcon>
-          <ListItemText>Add Telemetry Device</ListItemText>
+          <Typography variant="inherit">Add Telemetry Device</Typography>
         </MenuItem>
+        {props.devices?.length ? (
```

Comment:
Nit: Use `props.devices?.length &&` instead of ternary

Reply:
devices.length can resolve 0 in which case it will render zero in the menu, that's why I did ternary here.

Reply:
`{0 && (<MenuItem />)}` will render as a 0?

Reply:
![image](https://github.com/bcgov/biohubbc/assets/112989452/1aa502f3-b893-45ad-ada9-34953401c61f)
from react docs

Reply:
Ahh OK you got me.

I would just do `props.device?.length > 0` then, since this always resolves to `true` or `false`. Using a ternary and just throwing away the else branch seems like a bit of an anti-pattern to me. Using `length > 0 &&` immediately communicates that you want to render a list but only when it is non-empty.